R/zchunk_LA100.CDIAC_downscale_ctry.R

```diff
@@ -0,0 +1,131 @@
+#' module_energy_LA100.CDIAC_downscale_ctry
+#'
+#' Combine and downscale (back in time, for USSR and Yugoslavia) the CDIAC emissions and sequestration data.
+#'
+#' @param command API command to execute
+#' @param ... other optional parameters, depending on command
+#' @return Depends on \code{command}: either a vector of required inputs,
+#' a vector of output names, or (if \code{command} is "MAKE") all
+#' the generated outputs: \code{L100.CDIAC_CO2_ctry_hist}. The corresponding file in the
+#' original data system was \code{LA100.CDIAC_downscale_ctry.R} (energy level1).
+#' @details Combine and downscale (back in time, for USSR and Yugoslavia) the CDIAC emissions and sequestration data.
+#' @importFrom assertthat assert_that
+#' @importFrom dplyr filter mutate select
+#' @importFrom tidyr gather spread
+#' @author BBL April 2017
+module_energy_LA100.CDIAC_downscale_ctry <- function(command, ...) {
+  if(command == driver.DECLARE_INPUTS) {
+    return(c(FILE = "energy/CDIAC_CO2_by_nation",
+             FILE = "energy/CDIAC_Cseq_by_nation",
+             FILE = "energy/CDIAC_nation"))
+  } else if(command == driver.DECLARE_OUTPUTS) {
+    return(c("L100.CDIAC_CO2_ctry_hist"))
+  } else if(command == driver.MAKE) {
+
+    all_data <- list(...)[[1]]
+
+    # Load required inputs
+    CDIAC_CO2_by_nation <- get_data(all_data, "energy/CDIAC_CO2_by_nation")
```

Comment:
That looks like a mistake by the script the set up the new files; the original data system lines are
```R
CDIAC_CO2_by_nation <- readdata( "EMISSIONS_LEVEL0_DATA", "CDIAC_CO2_by_nation" )
CDIAC_Cseq_by_nation <- readdata( "EMISSIONS_LEVEL0_DATA", "CDIAC_Cseq_by_nation" )
CDIAC_nation <- readdata( "EMISSIONS_MAPPINGS", "CDIAC_nation" )
```

I'll move the files and update this PR.